### PR TITLE
fixes json logging in fluentd-elasticsearch image

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**: 
* fixes json logging in fluentd-elasticsearch image by downgrading fluent-plugin-kubernetes_metadata_filter plugin to version 2.0.0
* adds missing pos files to fluentd config 
* adds mapping type name 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
* https://github.com/helm/charts/issues/7168

**Special notes for your reviewer**:
* https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/145
* https://github.com/uken/fluent-plugin-elasticsearch/issues/442

**Release note**:

```release-note
fixes json logging in fluentd-elasticsearch image by downgrading fluent-plugin-kubernetes_metadata_filter plugin to version 2.0.0
```

### CODE REVIEW

File: cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml [hunk 1]
```diff
@@ -48,24 +48,24 @@ roleRef:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: fluentd-es-v2.2.0
+  name: fluentd-es-v2..0
```
Comment (coffeepac) [inline cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml]: please correct the version number.

**Comment by coffeepac** [comment]
@monotek can you split this into two PRs?  one for updating the image and one for updating the template?

this lets us rev the image without breaking the build as we still need to get outside help to manage the image build process.

**Comment by monotek** [comment]
used image version 2.2.0 again.
adding new pr for the template when image is build.

**Comment by neolit123** [comment]
/sig instrumentation
for the release note tool.

**Pull request merged**